hedera-node/hedera-config/src/main/java/com/hedera/node/config/sources/DynamicConfigSource.java

```diff
@@ -54,6 +55,17 @@ public String getValue(@NonNull final String s) throws NoSuchElementException {
         return properties.getProperty(s);
     }
 
+    @Override
+    public boolean isListProperty(@NonNull final String propertyName) throws NoSuchElementException {
+        return false;
```

Comment:
Is this always false? Or should be based on `propertyName`

Reply:
Currently, we're focusing on the YAML source, but we've created issues to add support for the new list properties in other sources as well. For now, the other sources continue to work in the original way.